test/models/tagging_test.rb

```diff
@@ -15,6 +15,10 @@
     tagging.must_respond_to(:taggable)
   end
 
+  it 'has an owner' do
+    tagging.owner.wont_be_nil
```

Comment:
yes it absolutely is, I just noticed some missing unit tests in the code coverage and put that in as a bonus.